examples/postgresql-ephemeral-template.json

```diff
@@ -5,7 +5,7 @@
     "name": "postgresql-ephemeral",
     "annotations": {
       "openshift.io/display-name": "PostgreSQL (Ephemeral)",
-      "description": "PostgreSQL database service, without persistent storage. For more information about using this template, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/9.5.\n\nWARNING: Any data stored will be lost upon pod destruction. Only use this template for testing",
+      "description": "PostgreSQL database service, without persistent storage. For more information about using this template, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/.\n\nWARNING: Any data stored will be lost upon pod destruction. Only use this template for testing",
```

Comment:
AFAIK this was meant to give container-specific instructions (real simplistic), thus reading `README.md` - usage - for direct usage of OpenShift; regardless of version(most ppl just use `latest`). IOW: I'd rather keep the 9.6.

Reply:
f.e. https://github.com/sclorg/postgresql-container/tree/master/9.5#environment-variables-and-volumes

Reply:
I have modified this in preparation for the last stages of image content generation (see #203 ) - eventually there will not be a `9.6` to directly link to and so the current link would result in a 404.

Reply:
Additionally, it is currently impossible to link to the 9.6 README anyway, without going through `latest/` which might change into a different version in the future. So it makes sense to me to just point users to the root.